docker/Dockerfile.worker

```diff
@@ -0,0 +1,31 @@
+FROM ufoym/deepo:py36-cpu
```

Comment:
Is there a reason you're using the same image for the notebook and the worker instead of two separate images or a shared base image?